Differ.xcodeproj/project.pbxproj

```diff
@@ -496,6 +506,25 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		6F04634327B0727E0062425C /* XCRemoteSwiftPackageReference "FastDiff" */ = {
```

Comment:
It looks like this imported Swift package is unused?

Reply:
Cleaned up the project file changes in https://github.com/tonyarnold/Differ/pull/84/commits/da3495a64e188dfd3de269ad355a41a1ac65ae0f 👍